### ISSUE TITLE
Test results is not displayed on Bitrise Report

### ISSUE THREAD

**Issue by moatazeldebsy** [HealthEngineAU/bitrise-step-bluepill-build-test-for-ios#2]
**Describe the bug**
I'm using Bluepill step with Bitrise but after the build succeeded I didn't find the report displayed and I don't know shall I pass a specific folder to Bitrise or what 

**Build information**
TargetName: demoreport
Scheme: demoreport
Workspace: Demo:Report.xcworkspace
Derived Data Path: build
Bluepill Logs Path: bluepill-output
Test Results Output Environment Variable: TEST_RESULTS

**Comment by reececomo** [comment]
Apologies for the delay in getting back to you - I haven't been able to reproduce this issue personally.

Is there any chance you could provide more details about what could have gone awry here? Screenshots/logs of config/build in action

**Comment by moatazeldebsy** [comment]
Hi @reececomo, actually we have a different problem that when using bluepill our app is opened displayed the splash screen only without running the tests in the scheme, can you help me in that?

**Comment by reececomo** [comment]
@moatazeldebsy Do you have `bluepill` installed locally?

I just saw from https://github.com/linkedin/bluepill/issues/377 you are running bluepill 5.1.1 (Xcode 11.1 + Swift 5.1).

I presume your UI/Unit Tests run normally otherwise.

* How many parallel simulators are you launching?
* Do you have any networking/expensive startup logic?
* Do you have any deferred app-launch startup logic?

**Comment by moatazeldebsy** [comment]
I tried with 6 devices and 1 device but the problem is the same, I asked the devs and they told me we have networking startup logic but not expensive, is there a problem from Bluepill with such these things.

**Comment by reececomo** [comment]
@moatazeldebsy I can't make any guarantees, but please check that using the `v0.3.0` release of the `HealthEngineAU/bitrise-step-bluepill-build-test-for-ios` step resolves your issue on Xcode 11.

-----

I overlooked that before - the most recent version of Bluepill that ships with the Bitrise StepLib version of our step is [Bluepill 4.1.1 (Xcode 10.2 compatible)](https://github.com/linkedin/bluepill/releases/tag/v4.1.1).

We have pushed an update for [Bluepill 5.0.0 (Xcode 11 compatible)](https://github.com/linkedin/bluepill/releases/tag/v5.0.0) support but we are waiting on Bitrise to [approve it](https://github.com/bitrise-io/bitrise-steplib/pull/2220).

Which you can use in your `bitrise.yml` file (in the Bitrise Workflow Editor menu) like this:

```yaml
    - git::https://github.com/HealthEngineAU/bitrise-step-bluepill-build-test-for-ios@0.3.0:
        title: iPhone SE @ 12.2
        inputs:
        - target_name: YourApp
        - scheme: YourApp
        - workspace: YourApp.xcworkspace
        - device_type: iPhone SE
        - test_result_env_var: IPHONE_SE_RESULTS
        - num_simulators: 3
        - retry_only_failed_tests: 'true'
```

We're looking at adding [Bluepill 5.1.0 (Xcode 11.1)](https://github.com/linkedin/bluepill/releases/tag/v5.1.0) support as soon as it's available via [Homebrew](https://formulae.brew.sh/formula/bluepill).
